### ISSUE TITLE
DB error 1054

### ISSUE THREAD

**Issue by AlfredSK** [friendica/friendica#8193]
### Observed behavior
I checked my log file and I found ~700 DB errors 1054. Maybe this is the cause of the issue I reported here? https://github.com/friendica/friendica/issues/8180#issuecomment-578854078

### Details of naughty behavior

``` 
2020-01-28T19:24:33Z index [ERROR]: DB Error {"code":1054,"error":"Unknown column 'abort' in 'field list'","callstack":"Database::insert, BaseRepository::insert, Notify::insert, notification, check_item_notification, check_user_notification, Item::insert, Processor::postItem","params":"INSERT INTO `notify` (`name`, `url`, `photo`, `uid`, `iid`, `parent`, `type`, `verb`, `otype`, `date`, `abort`) VALUES ('Steffen K9 \ud83d\udc30', 'https:\/\/libranet.de\/profile\/alfred', 'https:\/\/libranet.de\/photo\/3b2c1a18929163223620833d46bf017c-5.jpg?ts=1580226164', 33, 54291489, 54289238, 128, 'http:\/\/activitystrea.ms\/schema\/1.0\/tag', 'item', '2020-01-28 19:24:33', '')"} - {"file":"Database.php","line":714,"function":"e","uid":"2a9b3f","process_id":4875}
``` 

### Friendica version you encountered the problem
2020.03-dev

### Friendica source (git, zip)
git

### PHP version
7.2

### SQL version
10.1 MariaDB

**Comment by MrPetovan** [comment]
It's the cause of notifications not being stored and displayed.

**Comment by nupplaphil** [comment]
Fixed with #8186 as well :)
